src/lib/components/files/FileFolder.svelte

```diff
@@ -124,10 +126,20 @@
             }
         }
 
+
         :global(.svg-icon) {
             color: var(--warning-color);
             width: var(--icon-size-largest);
             height: var(--icon-size-largest);
         }
     }
+
```

Comment:
instead of using this, use the <Text singleLine> this should do the same thing without the need for a new class if possible